css/index.css

```diff
@@ -3,13 +3,282 @@ This imports the `reset.css` file */
+}
+
+.title-container {
+  margin: 20px auto;
+  border-radius: 8px;
+  width: 350px;
+}
+
+.title,
+.theme-text,
+.theme-num,
+.theme-color-container {
+  color: var(--text-color);
+  font-weight: 700;
+}
+
+.title {
+  font-size: 25px;
+  margin-left: 25px;
+  padding-top: 10px;
+}
+
+.theme-container {
+  display: flex;
+  flex-direction: row;
+  margin-top: -20px;
+}
+
+.theme-text {
+  margin: 19px 10px 0 55%;
+  font-size: 10px;
+}
+
+.theme-feature,
+.theme-num,
+.theme-color-container,
+.container {
+  display: flex;
+}
+
+.theme-feature {
+  flex-direction: column;
+  margin: 15px 0 0 10px;
+}
+
+.theme-num {
+  flex-direction: row;
+  margin: -5px 0 0 -9px;
+}
+
+.num {
+  margin: -20px 0 0 -1px;
+}
+
+.theme-color-container {
+  flex-direction: row;
+  margin-left: 5px;
+  height: 25px;
+  width: 70px;
+  border-radius: 90px;
+  background-color: var(--checkbox-underlay-color);
+}
+
+.container {
+  position: relative;
+  padding-left: 14px;
+  margin-bottom: 12px;
+  font-size: 12px;
+  font-weight: 700;
+}
+
+/* Create a custom radio button */
+.checkmark-one,
+.checkmark-two,
+.checkmark-three,
+.container input {
+  position: absolute;
+  top: 23px;
+  height: 18px;
+  width: 17px;
+  border-radius: 50%;
+}
+
+/* Hide the browser's default radio button */
+.container input {
+  opacity: 0;
+  cursor: pointer;
+}
+
+.checkmark-one {
+  left: 2px;
+}
+
+.checkmark-two {
+  left: 9px;
+}
+
+.checkmark-three {
+  left: 13px;
+}
+
+/* When the radio button is checked, add a blue background */
+.container input:checked ~ .checkmark-one {
+  background-color: var(--checkmark-one-background-color);
+}
+
+.container input:checked ~ .checkmark-two {
+  background-color: var(--checkmark-two-background-color);
+}
+
+.container input:checked ~ .checkmark-three {
+  background-color: var(--checkmark-three-background-color);
+}
+
+.result-container {
+  width: 320px;
+  height: 90px;
+  margin: auto;
+  background-color: var(--result-background-color);
+  border-radius: 5px;
+}
+
+.result {
+  margin-left: 60%;
+  font-size: 32px;
+  margin-top: 30px;
+  padding-top: 27px;
+  color: var(--text-color);
+}
+
+.button-container {
+  width: 320px;
+  margin: auto;
+  background-color: var(--checkbox-underlay-color);
+  border-radius: 5px;
+}
+
+.buttons-signs {
+  display: grid;
+  margin: 10px;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+}
+
+.btn {
+  background-color: var(--button-background-color);
+  border: none;
+  color: var(--button-text-color);
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 32px;
+  margin: 15% 0 12px 6px;
+  width: 57px;
+  height: 55px;
+  cursor: pointer;
+  box-shadow: 0 5px var(--button-boxshadow);
+  font-weight: 700;
+  border-radius: 5px;
+}
+
+.del-btn {
+  background-color: var(--del-background-color);
+  color: var(--neutral-color);
+  font-size: 20px;
+  box-shadow: 0 5px var(--delsign-boxshadow);
+}
+
+.buttons-text {
+  display: flex;
+  flex-direction: row;
```

Comment:
this might not be necessary as you should have the flex row direction on default

Reply:
Fixed